app/models/episode.rb

```diff
@@ -281,4 +281,21 @@ def published_or_released_date
   def ready_transcript
     transcript if transcript&.status_complete?
   end
+
+  def head_request(uri_str = enclosure_url(podcast.default_feed), redirects = 0)
```

Comment:
If this method has a lot on common with the `RedirectPrefixValidator#head_request`... you could try to refactor into a shared library?

Probably a class method on a model concern (can a validator include a concern?)  And it would have to take in args: the url, max jumps allowed, and timeout.

In no way required!  Just a thought.